pontoon/base/static/js/translate_old.js

```diff
@@ -1235,6 +1235,10 @@ var Pontoon = (function (my) {
             name = project.html(),
             slug = project.data('slug');
 
+        if (slug !== 'pontoon-intro' && !self.user.email) {
```

Comment:
We don't have a better solution for this right now so don't worry about it, but someday I'd love to stop hardcoding pontoon-intro stuff everywhere (or id=1 stuff and so on).